packages/eas-cli/src/submissions/ios/IosSubmitter.ts

```diff
@@ -23,66 +21,29 @@ interface ResolvedSourceOptions {
   appSpecificPassword: string;
 }
 
-class IosSubmitter {
-  constructor(private ctx: IosSubmissionContext, private options: IosSubmissionOptions) {}
+class IosSubmitter extends BaseSubmitter<IosSubmissionContext, IosSubmissionOptions> {
+  protected readonly appStoreName: string = 'Apple TestFlight';
+
+  constructor(ctx: IosSubmissionContext, options: IosSubmissionOptions) {
+    super(SubmissionPlatform.iOS, ctx, options);
+  }
 
   async submitAsync(): Promise<void> {
     const resolvedSourceOptions = await this.resolveSourceOptions();
     const submissionConfig = await this.formatSubmissionConfigAsync(
       this.options,
       resolvedSourceOptions
     );
-    await this.startSubmissionAsync(submissionConfig, this.ctx.commandFlags.verbose);
-  }
-
-  // TODO: Temp - change public to private
-  public async startSubmissionAsync(
-    submissionConfig: IosSubmissionConfig,
-    verbose: boolean = false
-  ): Promise<void> {
-    const scheduleSpinner = ora('Scheduling submission').start();
-    let submissionId: string;
-    try {
-      submissionId = await SubmissionService.startSubmissionAsync(
-        SubmissionPlatform.iOS,
-        submissionConfig.projectId,
-        submissionConfig
-      );
-      scheduleSpinner.succeed();
-    } catch (err) {
-      scheduleSpinner.fail('Failed to schedule submission');
-      throw err;
-    }
 
-    let submissionCompleted = false;
-    let submissionStatus: SubmissionStatus | null = null;
-    let submission: Submission | null = null;
-    const submissionSpinner = ora('Submitting your app to Apple TestFlight').start();
-    try {
-      while (!submissionCompleted) {
-        await sleep(DEFAULT_CHECK_INTERVAL_MS);
-        submission = await SubmissionService.getSubmissionAsync(
-          submissionConfig.projectId,
-          submissionId
-        );
-        submissionSpinner.text = IosSubmitter.getStatusText(submission.status);
-        submissionStatus = submission.status;
-        if (submissionStatus === SubmissionStatus.ERRORED) {
-          submissionCompleted = true;
-          process.exitCode = 1;
-          submissionSpinner.fail();
-        } else if (submissionStatus === SubmissionStatus.FINISHED) {
-          submissionCompleted = true;
-          submissionSpinner.succeed();
-        }
-      }
-    } catch (err) {
-      submissionSpinner.fail(IosSubmitter.getStatusText(SubmissionStatus.ERRORED));
-      throw err;
-    }
-
-    await displayLogs(submission, submissionStatus, verbose);
+    printSummary(
+      submissionConfig,
+      'iOS Submission Summary',
+      SummaryHumanReadableKeys,
+      SummaryHumanReadableValues
+    );
+    // await this.startSubmissionAsync(submissionConfig, this.ctx.commandFlags.verbose);
```

Comment:
Is this commented intentionally?

Reply:
Yes, as this is only chained PR to a draft, I temporarily commented this out to do some manual tests and don't trigger remote Submission Service unnecessarily. It's gonna be of course uncommented later.